sagemaker-fundamentals/create-training-job/create_training_job_encrypt_static_assets.ipynb

```diff
@@ -0,0 +1,1227 @@
+{
```

Comment:
<p>Makes more sense to use "allow yourself" versus "allow myself" for the customer's point of view. (3 uses of "myself")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>